### PR TITLE
Adding image operations documentation

### PR DESCRIPTION
**Pull request by pappacena**
Image operations documentation and examples.

image

### CODE REVIEW

File: docs/image-operations.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+frame.save_image("docs/images/cereal-ops/cereal-color-2.0.jpeg")
+```
+
+![Cereal image](../images/cereal-ops/cereal.jpeg)
+![Cereal low color adjustment](../images/cereal-ops/cereal-color-0.1.jpeg)
+![Cereal high color adjustment](../images/cereal-ops/cereal-color-2.0.jpeg)
+
+## Contrast
+
+Similar to color, you can adjust the contrast. The factor parameter must be 1.0 for no change, less than 1.0 for less intensity and greater than 1.0 for more intensity.
```
Comment (lai-bianca) [inline docs/image-operations.md:82]: Reworded:
You can adjust the contrast intensity:

- No change: 1.0
- Less intensity: Less than 1.0
- More intensity: Greater than 1.0

See the following examples.

File: docs/image-operations.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+```
+
+
+![Cereal image](../images/cereal-ops/cereal.jpeg)
+![Cereal low contrast adjustment](../images/cereal-ops/cereal-contrast-0.1.jpeg)
+![Cereal high contrast adjustment](../images/cereal-ops/cereal-contrast-2.0.jpeg)
+
+## Brightness
+
+To adjust brightness, use the same factor parameter: 1.0 for no change, less than 1.0 for less intensity and greater than 1.0 for more intensity.
```
Comment (lai-bianca) [inline docs/image-operations.md:105]: Reworded:
You can adjust the brightness intensity:

- No change: 1.0
- Less intensity: Less than 1.0
- More intensity: Greater than 1.0

See the following examples.

File: docs/image-operations.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+```
+
+
+![Cereal image](../images/cereal-ops/cereal.jpeg)
+![Cereal low brightness adjustment](../images/cereal-ops/cereal-brightness-0.1.jpeg)
+![Cereal high brightness adjustment](../images/cereal-ops/cereal-brightness-2.0.jpeg)
+
+## Sharpness
+
+Adjusting sharpness follows the same pattern as color, brightness and contrast: 1.0 for no change, less than 1.0 for less intensity and greater than 1.0 for more intensity.
```
Comment (lai-bianca) [inline docs/image-operations.md:128]: Reworded:
You can adjust the sharpness intensity:

- No change: 1.0
- Less intensity: Less than 1.0
- More intensity: Greater than 1.0

See the following examples.

**Review by lai-bianca — COMMENTED** [review]
See in-line comments.

**Review by lai-bianca — APPROVED** [review]

**Review by camiloaz — APPROVED** [review]

**Pull request merged**